galaxy/api/internal/views/search.py

```diff
@@ -133,6 +145,16 @@ def get_order_by(self, request):
                 f'{repr(param)} is not a valid ordering parameter value.')
         return param, order
 
+    def get_format_type(self, request):
```

Comment:
```
def get_format_type(self, request):
    format = request.query_params.get('type')  # Default is already None.
    if format is None:
        return None
    format = format.lower()  # Not sure though about allowing case-insensitive values.
    if format not in ALLOWED_CONTENT_TYPES:
            raise exceptions.ValidationError(
                f'"{format}" is not a valid format type.')
    return format
```

Also it's a bit confusing that constant is called `ALLOWED_CONTENT_TYPES` but method gets `format_type`

Reply:
Also it would've been easier to move this logic to SearchRequestSerializer
ChoiceField with default value will do.